homeassistant/components/vizio/media_player.py

```diff
@@ -483,3 +484,11 @@ async def async_set_volume_level(self, volume: float) -> None:
                 num = int(self._max_volume * (self._attr_volume_level - volume))
                 await self._device.vol_down(num=num, log_api_exception=False)
                 self._attr_volume_level = volume
+
+    async def async_media_play(self) -> None:
+        """Play whatever media is currently active."""
+        await self._device.play(log_api_exception=False)
+
+    async def async_media_pause(self) -> None:
+        """Pause whatever media is currently active."""
+        await self._device.pause(log_api_exception=False)
```

Comment:
Why does one of these set `log_api_exception` to True and the other to False?

Reply:
I'm not sure what you mean; all instances of the `log_api_exception` parameter are set to `False` in this file.

Reply:
You're right, I have no idea what I meant 🙈